shell/platform/tizen/channels/key_event_channel.cc

```diff
@@ -8,25 +8,29 @@
 
 #include "flutter/shell/platform/tizen/tizen_log.h"
 
-static constexpr char kChannelName[] = "flutter/keyevent";
+namespace flutter {
 
-static constexpr char kKeyMapKey[] = "keymap";
-static constexpr char kKeyCodeKey[] = "keyCode";
-static constexpr char kScanCodeKey[] = "scanCode";
-static constexpr char kTypeKey[] = "type";
-static constexpr char kModifiersKey[] = "modifiers";
-static constexpr char kToolkitKey[] = "toolkit";
-static constexpr char kUnicodeScalarValuesKey[] = "unicodeScalarValues";
+namespace {
 
-static constexpr char kKeyUp[] = "keyup";
-static constexpr char kKeyDown[] = "keydown";
-static constexpr char kGtkToolkit[] = "gtk";
-static constexpr char kLinuxKeyMap[] = "linux";
+constexpr char kChannelName[] = "flutter/keyevent";
```

Comment:
Why do you remove static? Because of namespaces?

Reply:
No, they are functionally equivalent (https://google.github.io/styleguide/cppguide.html#Internal_Linkage), but I think using unnamed namespaces is more C++ way and makes the code cleaner.

Reply:
```c++
namespace {
```
I passed it without thinking, but it has sense!